### ISSUE TITLE
[Bug]: ValidationError in AccountDocument

### ISSUE THREAD

**Issue by sergioov** [alpacahq/alpaca-py#303]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When getting an account by id using broker_client.get_account_by_id({account_id}) pydantic data validation fails on document model creation. 

The document type for "Account Application" sub_type is ""
Ex. 
"document_type": "",
"document_sub_type": "Account Application",

The enum in DocumentType is missing a "" type to accept it as valid. 

### Expected Behavior

When I do a account = broker_client.get_account_by_id({account_id}) should return a valid account model object instance. 

### SDK Version I encountered this issue in

Name: alpaca-py
Version: 0.8.2

### Steps To Reproduce

```markdown
In sandbox environment with broker api
trying to get an account created in sandbox
The returned data contains a documents list which includes a document with document_type = "" and document sub_type = "Account Application"

use account = broker_client.get_account_by_id({account_id}) for an account_id with an Account Application document.
```

### Filled out the Steps to Reproduce section?

- [X] I have entered valid steps to reproduce my issue or have attached a minimally reproducible case in code that shows my issue happening; and understand that without this my issue will be flagged as invalid and closed after 30 days.

**Comment by gageorsburn** [comment]
This is a dupe / related to #273

**Comment by haxdds** [comment]
Should be resolved with #308
